File: rb/test/exists_test.rb
# YahooFinance SDK exists test

require "minitest/autorun"
require_relative "../YahooFinance_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = YahooFinanceSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- YahooFinance SDK exists test

local sdk = require("yahoo-finance_sdk")

describe("YahooFinanceSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
